src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,67 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Represents a user interface of the chat robot.
+ */
+public class Ui {
+    public void showLoadingError() {
+        System.out.println("Generating new chat session...");
+    }
+
+    public void showSavingError() {
```

Comment:
I like the descriptive naming sense for your functions, definitely helps with the clarity of what those functions do!